uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/TasksassignmentsTestTrait.php

```diff
@@ -23,23 +24,27 @@ protected function createContact($params) {
   }
 
   /**
-   * Creates single (Individuals) contact from the provided data.
+   * Create case type
    *
-   * @param array $params should contain first_name and last_name
-   * @return int return the contact ID
-   * @throws \CiviCRM_API3_Exception
+   * @param array $params should contain 'name'
```

Comment:
Please start the $param description with an upper case letter in order to make it easier to spot where it starts: Should contain 'name'. The same applies to the doc blocks on the other methods